03-transactions.md

```diff
@@ -120,14 +120,14 @@ This output sends funds to a HTLC-timeout transaction after the HTLC timeout, or
             OP_DROP 2 OP_SWAP <localkey> 2 OP_CHECKMULTISIG
         OP_ELSE
             # To you with preimage.
-            OP_HASH160 <ripemd-of-payment-hash> OP_EQUALVERIFY
+            OP_HASH160 <ripemd-of-payment_hash> OP_EQUALVERIFY
```

Comment:
Technically, I was trying to replace all field names (payment_hash is a field of update_add_htlc).  But I think we should actually put <RIPEMD160(payment_hash)> here.  Will add a new commit...